pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -753,3 +774,24 @@ func getEksCloudwatchLogConfig(client client.Client, interval int32, region, gro
 		FetchInterval: interval,
 	}, nil
 }
+func getSysLogCertificate(client client.Client) (certificatemanagement.CertificateInterface, error) {
+	cm := &corev1.ConfigMap{}
+	cmNamespacedName := types.NamespacedName{
+		Name:      render.SyslogCAConfigMapName,
+		Namespace: common.OperatorNamespace(),
+	}
+	if err := client.Get(context.Background(), cmNamespacedName, cm); err != nil {
+		if errors.IsNotFound(err) {
+			log.Info(fmt.Sprintf("ConfigMap %q is not found. Assuming internet trusted CA certificate", render.SyslogCAConfigMapName))
```

Comment:
Maybe rephrase to "ConfigMap %q is not found, assuming Syslog's certificate is signed by a publicly trusted CA"

Reply:
Done